Lesson34/selcan/lesson34-app/src/App.js

```diff
@@ -0,0 +1,42 @@
+import { List } from './components/List';
+import { Navbar } from './components/Navbar';
+import { PerformanceState } from './components/PerformanceState';
+import { useState } from 'react';
+import { todoData } from './data';
+import { UserProvider } from './userContext';
+import { Account } from './components/Account';
+import Login from './components/Login';
+import './App.css';
+
+// TODO: Create a new Component that contains log in form, so that user can type username and password and click submit button to log in.
+// If user is logged in, show Account component. 
+// If user is logged out, show Log In component.
+
+export const App = () => {
+	const [todos, setTodos] = useState(todoData);
+
+	const loggedOutUser = {
+		isLoggedInUser: false
+	};
+
+	return (
+
+		<UserProvider initialState={loggedOutUser}>
+			<div className='container'>
+				<Navbar />
+				<Login />
```

Comment:
There're two requirements that are still not met:

>  In the App component, if user is logged in, show Account component.
- In the App component, if user is logged out, show Log In component.

Consume user data from the Context: 

```
const user = useContext(UserContext);

```

then use it to decide if Login and Account components should be on the screen or not.